adapters/ix/ix.go

```diff
@@ -275,8 +425,10 @@ func getMediaTypeForBid(bid openrtb2.Bid, impMediaTypeReq map[string]openrtb_ext
 // Builder builds a new instance of the Ix adapter for the given bidder with the given config.
 func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
 	bidder := &IxAdapter{
-		URI:         config.Endpoint,
-		maxRequests: 20,
+		URI:                    config.Endpoint,
+		maxRequests:            100,
```

Comment:
Each request is made independently. Your adapter latency may increase or you may starve connections with a significant increase to the amount of "fan out" requests allowed per incoming request. I highly suggest you make this configurable using the adapter extra info such that hosts have the option of lowering this value instead of just disabling this adapter outright if performance issues are encountered.